### PR TITLE
Added ActiveRecord metrics to Yabeda Prometheus Exporter

### PR DESCRIPTION
**Pull request by srijitamukherjee**
**What this PR does / why we need it**:

> Adds Active Record Metrics to Yabeda Prometheus Exporter

**Which issue(s) this PR fixes** 

>https://issues.redhat.com/browse/THREESCALE-5698

**Verification steps** 

> Clone this PR
> Run Porta
> Verify the metrics from [http://localhost:9394/yabeda-metrics](http://localhost:9394/yabeda-metrics)

**Special notes for your reviewer**:
Screenshot from 2020-11-17 19-18-17

### CODE REVIEW

File: lib/prometheus/active_record.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Yabeda.configure do
+  group :rails_connection_pool do
+    size = gauge :size, comment: 'Size of the connection pool'
+    connections = gauge :connections, comment: 'Number of connections in the connection pool'
```
Comment (guicassolato) [inline lib/prometheus/active_record.rb:6]:
```suggestion
    connections = gauge :connections, comment: 'Number of connections in the connection pool', tags: %i[state]
```

And you'll able to use the labels again to separate "busy", "dead", "idle".

**Comment by srijitamukherjee** [comment]
@guicassolato Can you please review this again ? Thanks

**Review by hallelujah — APPROVED** [review]
Looks good to me

**Review by guicassolato — APPROVED** [review]
Nice work! Thanks

**Comment by hallelujah** [comment]
Sorry, it seems your work did not get enough attention from us.
Thank you for making this project better.

We are going to close it as it was stale for too long.

**Pull request closed without merging**